chart/ci/test-values.yaml

```diff
@@ -0,0 +1,44 @@
+---
+replicaCount: 2
+
+fullnameOverride: xnat-core
+
+image:
+  name: xnat-core
+  tag: latest
+
+imageCredentials:
+  enabled: true
+  registry: ghcr.io
+
+postgresql:
+  fullnameOverride: xnat-core-postgresql
```

Comment:
setting `fullnameOverride` is necessary so the name of the secret created in `chart/ci/manifests/secrets.yaml` matches what the CNPG Cluster chart expects